src/isolate-duplicates/index.js

```diff
@@ -1,3 +1,63 @@
-function isolateDuplicates(text) {}
+function isolateDuplicates(text) {
+  // handle wrong input
+  if (typeof text !== "string") {
+    throw "Please enter a valid string";
```

Comment:
Prefer to `throw Error(...)` rather than strings. Helps preserve the stack trace for debugging.